packages/dx-react-grid/docs/reference/table-group-header.md

```diff
@@ -0,0 +1,60 @@
+# TableGroupHeader Plugin Reference
+
+A plugin that renders the nested header cells.
+
+## User Reference
+
+### Dependencies
+
+- [Table](table.md)
+- [TableHeaderRow](table-header-row.md)
+- [TableEditColumn](table-edit-column.md) [Optional]
+- [TableSelection](table-selection.md) [Optional]
+- [DragDropProvider](drag-drop-provider.md) [Optional]
+- [TableColumnResizing](table-column-resizing.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+cellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a parent cell.
+rowComponent | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders row for parent cells.
+headerCellComponent | ComponentType&lt;[TableHeaderRow.CellProps](table-header-row.md#tableheaderrowcellprops)&gt; | | A component that renders a header cell.
+columnGroups | Array&lt;[TableGroupHeader.ColumnGroups](#tablegroupheadercolumngroups)&gt; | | Specifies the nested columns for implement multiple levels of columns in your table header.
+
+## Interfaces
+
+### TableGroupHeader.ColumnGroups
+
+Name | Properties | Description
+-----|------------|------------
+columnName? | string | The name of the column.
+title? | string | The title of the parent column.
+nested? | Array&lt;[TableGroupHeader.ColumnGroups](#tablegroupheadercolumngroups)&gt; | The nested columns of current title.
+
+## Plugin Components
+
+Name | Properties | Description
+-----|------------|------------
+TableGroupHeader.Cell | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a parent cell.
+TableGroupHeader.Row | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders row for parent cells.
+TableGroupHeader.HeaderCell | ComponentType&lt;[TableHeaderRow.CellProps](table-header-row.md#tableheaderrowcellprops)&gt; | | A component that renders a header cell.
```

Comment:
Something strange with this component. It doesn't seem to render header. Is it true?

Reply:
It also has a different interface

Reply:
Maybe better set `HeaderCellWrapper` name to this component?

Reply:
We use the `container` word in such cases

Reply:
But I am not sure how we should call this component